dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/tei/query/context/querybuilder/DataElementQueryBuilder.java

```diff
@@ -56,6 +55,11 @@
 @Service
 @org.springframework.core.annotation.Order(999)
 public class DataElementQueryBuilder implements SqlQueryBuilder {
+
+  @Getter
+  private final List<Predicate<DimensionIdentifier<DimensionParam>>> headerFilters =
```

Comment:
Should we call it just `headers`? He have the impression that it might apply some filtering, which not the case...

Reply:
I understand it's a bit misleading and we might find a better name, but this is a collection of predicates, which returns true if the header is one to be taken into consideration by the current queryBuilder.
In this case, it's a test to check if headers refer to a data element.